src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1741,13 +1742,41 @@ source = do
 linkTitle :: MarkdownParser String
 linkTitle = quotedTitle '"' <|> quotedTitle '\''
 
-link :: MarkdownParser (F Inlines)
-link = try $ do
+wikilinkInfo :: MarkdownParser (F Inlines, String)
+wikilinkInfo = do
+  string "[["
+  (_, raw) <- withRaw $ charsInBalancedBrackets 2
+  guard $ not $ null raw
+  let deunder c = if c == '_' then ' ' else c
+      deiki s = map deunder $ (reverse . takeWhile (/= '/') . reverse) $ s
+      rawstr = (reverse . drop 2 . reverse) raw
```

Comment:
Stripping the trailing `]]` is a job probably best done using by Parsec.  See note above.